etc/adminhtml/system.xml

```diff
@@ -58,6 +58,10 @@
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                     <comment><![CDATA[Yes - order gets created in the store prior to transaction authorization<br />No - transaction has to be authorized before order is created in the store]]></comment>
                 </field>
+                <field id="product_page_checkout" translate="label" type="select" sortOrder="80" showInDefault="1" showInWebsite="1" showInStore="0">
+                    <label>Enable product page checkout (beta)</label>
```

Comment:
Can you slightly reword this label?
`(BETA) Enable Product Page Checkout`

Reply:
Done